sql_script/create_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS user (
+    user_id VARCHAR(64) PRIMARY KEY,
+    user_type VARCHAR(32) NOT NULL,
+    name VARCHAR(64)
+) ENGINE=InnoDB CHARACTER SET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+CREATE TABLE IF NOT EXISTS accounting (
+    id INT UNSIGNED PRIMARY KEY,
```

Comment:
miss AUTO_INCREMENT
`id INT UNSIGNED NOT NULL PRIMARY KEY AUTO_INCREMENT`

Reply:
You're right and I've added.